src/clj/game/cards/resources.clj

```diff
@@ -1545,6 +1545,26 @@
                :runner-turn-ends {:effect (effect (update! (dissoc card :server-target)))}}
       :abilities [ability]})
 
+   "Slipstream"
+    {:abilities [{:label "[Trash]: Choose a piece of ice protecting a central server in the same position as the passed ice and approach that ice."
+                     :req (req (and (:run @state)))
+                     :effect (effect (req (let [icepos (ice-index state current-ice)]
+                                            (resolve-ability state side
+                                             {:prompt (msg "Choose a piece of ICE protecting a central server at position " icepos)
+                                              :choices {:req #(and (is-central? (second (:zone %)))
+                                                                   (ice?)
+                                                                   (=(icepos (ice-index state %))))}
+                                              :msg (msg "move to " (card-str state target))
+                                              :effect (req (let [dest (second (:zone target))
+                                                                 tgtndx (ice-index state target)]
+                                                             (swap! state update-in [:run]
+                                                                    #(assoc % :position tgtndx :server [dest]))
+                                                             (trash state side card {:cause :ability-cost})
```

Comment:
Same comment as the Jackpot PR: this not an ability cost. It should, however, be unpreventable because of the "If you do" clause.